flake.nix

```diff
@@ -23,7 +23,7 @@
       lib = {
         hm = (import ./modules/lib/stdlib-extended.nix nixpkgs.lib).hm;
         homeManagerConfiguration = { modules ? [ ], pkgs, lib ? pkgs.lib
-          , extraSpecialArgs ? { }, check ? true
+          , extraSpecialArgs ? { }, check ? true, disableNixpkgsModule ? true
```

Comment:
Why the negation? `useNixpkgsModule`. Also it should default to whatever its value is now, to preserve backwards compatibility.